src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,30 +16,37 @@ import './WebexLocalMedia.scss';
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
-export default function WebexLocalMedia({meetingID}) {
+export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
 
   const classBaseName = 'local-media';
-  const classObjects = {};
+  const mainClasses = {};
 
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}`] = true;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}-desktop`] = width >= PHONE_LARGE;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-no-media`] = localVideo === null;
+  mainClasses[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}`] = true;
```

Comment:
I know this change was done on a previous PR, but I would like to go back to the way the code looked before because I think that's much more readable (and the reason I felt comfortable with using the `classnames` library). The same can be achieved with computed property names.

```js
const baseCSSClass = `${WEBEX_COMPONENTS_CLASS_PREFIX}-local-media`;
const cssClasses = classNames({
  [baseCSSClass]: true,
  [`${baseCSSClass}-desktop`]: width >= PHONE_LARGE,
  ...,
 className
});
```
This is non-blocking for this since the code already exists, but hope is ok to open a PR later to refactor it.

Reply:
I honestly had no idea that was an option.

![img](https://media0.giphy.com/media/83QtfwKWdmSEo/giphy.gif)